testing/server-item-renderer-with-debug-ui.tsx

```diff
@@ -9,7 +9,7 @@ import * as Perseus from "../packages/perseus/src/index";
 import {keScoreFromPerseusScore} from "../packages/perseus/src/util/scoring";
 
 import KEScoreUI from "./ke-score-ui";
-import SideBySide from "./side-by-side";
+import SideBySide from "./split-view";
```

Comment:
Absolutely NOT a blocker for landing, but I think this is supposed to be `SplitView`

Reply:
Great catch! I can definitely fix that with a quick micro PR :) 

Thank you!